### DISCUSSION TITLE
bootstrap suggestion

### DISCUSSION THREAD

**Discussion by Jonatas-Felipe** [twbs/bootstrap#33618] [Ideas]
Hello I created some classes to complement the bootstrap, classes that I missed a lot during the development of my projects, all based on the bootstrap itself, if you can take a look at this attachment.

Thanks for listening.

[bootstrap suggestion.zip](https://github.com/twbs/bootstrap/files/6304339/bootstrap.suggestion.zip)

**Comment by florianlacreuse** [comment]
Hi,

I suggest you to open a pull request, or at least provide a detailed explanation in this thread.
